pkg/network/encoding/usm_windows_test.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
this file is owned by us, but it should be owned by WKIT
please make the relevant changes to CODEOWNERS file, and ask for @richl-dd review on the windows files